src/main/java/uk/gov/ons/census/casesvc/service/UacService.java

```diff
@@ -73,6 +73,13 @@ private UacQidLink buildUacQidLink(Case linkedCase, UUID batchId, String uac, St
     return uacQidLink;
   }
 
+  public UacQidLink buildCCSUacQidLink(int questionnaireType) {
+    UacQidLink uacQidLink = buildUacQidLink(null, questionnaireType);
```

Comment:
why pass `null` here when there's a place for `Case` to be passed in? Add `Case` as a parameter or lose this method completely because it adds no value.

Reply:
done